lib/levelup.js

```diff
@@ -55,6 +56,23 @@ function LevelUP (db, options, callback) {
   this._db = db
   this.db = new DeferredLevelDOWN(db)
   this.open(callback)
+
+  // Create manifest based on deferred-leveldown's
+  this.supports = supports(this.db.supports, {
+    status: false,
+    deferredOpen: true,
+    openCallback: true,
+    promises: true,
+    streams: true
+  })
+
+  Object.keys(this.supports.additionalMethods).forEach(function (method) {
+    this[method] = this[method] || function () {
+      // TODO: uhm. if the manifest is based on deferred-leveldown's,
+      // should we also call the methods via deferred-leveldown?
```

Comment:
This is tricky.

Reply:
Can work if we agree that `deferred-leveldown` itself must not add methods to its `additionalMethods`; it should inherit that from its input db (same as `levelup`).